src/screens/RegisterTokenManual.js

```diff
@@ -57,12 +57,14 @@ class RegisterTokenManual extends React.Component {
       return;
     }
 
-    const ret = hathorLib.tokens.validateTokenToAddByConfigurationString(this.state.configString);
-    if (ret.success) {
-      this.setState({ token: ret.tokenData, errorMessage: '' });
-    } else {
-      this.setState({ errorMessage: ret.message });
-    }
+    const promise = hathorLib.tokens.validateTokenToAddByConfigurationString(
```

Comment:
I wonder if we should show a spinner, as this calls the network and could take a bit. Otherwise, there's no feedback to the user.

Reply:
Makes sense, added the loading.